shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -122,6 +121,7 @@
     public static final String CACHE_USED_SIZE_BYTES = PREFIX + "segmentstore.cache.used_size_bytes";       // Gauge
     public static final String CACHE_ALLOC_SIZE_BYTES = PREFIX + "segmentstore.cache.allocated_size_bytes"; // Gauge
     public static final String CACHE_GENERATION_SPREAD = PREFIX + "segmentstore.cache.gen";                 // Histogram
+    public static final String CACHE_POLICY_DURATION = PREFIX + "segmentstore.cache.policy_duration_ms";    // Timer
```

Comment:
Let's name this `segment store.cache.manager_iteration_duration_ms` - it's more indicative of what it does. Hopefully not too long of a name.